addons/purchase/static/src/js/purchase_portal_connect_modal.js

```diff
@@ -0,0 +1,11 @@
+function getHash() {
+    const hash = window.location.hash;
+    return hash ? hash.split('?').at(0) : null;
+}
+$(document).ready(function() {
+    const hash = getHash();
+    if(hash === '#portal_connect_software_modal') {
+        const modalButton = document.querySelector("#portal_connect_software_modal_btn");
+        modalButton.click();
+    }
+});
```

Comment:
Seems a bit hacky. It's just to force the click right?

Reply:
We need to open the modal in the page at load time if the portal page is opened from the link in the PO PDF (part of the requirements). This is done by simulating the click, which in turn runs `bootstrap` code to toggle the modal.

Reply:
Refused to execute script from 'http://localhost:8069/purchase/static/src/js/portal_connect_software_modal.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.